proxy/types.go

```diff
@@ -171,6 +202,34 @@ func uniqueKeyMevSendBundle(b *MevSendBundleArgs, hash hash.Hash) {
 	_, _ = hash.Write(b.Metadata.Signer.Bytes())
 }
 
+func (b *MevSendBundleArgs) Validate() (common.Hash, error) {
+	if len(b.Body) == 0 {
+		return common.Hash{}, ErrBundleNoTxs
+	}
+	hash := sha3.NewLegacyKeccak256()
+	if err := hashMevSendBundle(b, hash); err != nil {
+		return common.Hash{}, err
+	}
+	return common.BytesToHash(hash.Sum(nil)), nil
+}
+
+func hashMevSendBundle(b *MevSendBundleArgs, hash hash.Hash) error {
```

Comment:
Lets add depth validation here too

Reply:
Added in https://github.com/flashbots/tdx-orderflow-proxy/pull/4/commits/0bbba62e1b0a429e22f13a0c314f720ce5d7b72e